### DISCUSSION TITLE
Berechnung CO2-Budget: Ernährung und Konsum wirklich pauschal abziehen?

### DISCUSSION THREAD

**Discussion by jayAitch256** [codeformuenster/klimawatch#133] [General]
Auf der [Erklärungsseite zur Berechnung des CO2-Budgets](https://klimawatch.de/paris-limits/) steht:

> Warum haben wir 40% abgezogen? Das liegt daran, dass in der Bilanz der Stadt weder individueller Konsum noch die Ernährung der BürgerInnen eingerechnet ist. Lediglich Strom, Wärme und Verkehr sind in der städtischen Energiebilanz berücksichtigt. Da ca. 40 % eines individuellen CO2-Fußabdrucks für Ernährung und Konsum draufgehen (diese Zahl haben wir vom CO2-Rechner des Umweltbundesamts), müssen wir diese für eine Vergleichbarkeit mit den städtischen Bilanzen also abziehen.

Die Argumentation, dass Ernährung und Konsum nicht in die städtische Energiebilanz einfließen, stimmt genau genommen nur halb. Wenn ich regionale Produkte konsumiere, schlagen die ja sehr wohl in der Bilanz zu Buche, oder? (Strom-/Wärmeverbrauch des produzierenden Unternehmens + Verkehr). 40% sind also, vermute ich, zu hoch gegriffen.

Gegenvorschlag: Nicht den Verbrauch, sondern die Emissionen betrachten. In der [BISKO-Methode](https://www.ifeu.de/fileadmin/uploads/BISKO_Methodenpapier_kurz_ifeu_Nov19.pdf), die von den allermeisten Kommunen inzwischen zum Erstellen von THG-Bilanzen angewendet wird, fehlen meistens die nichtenergetischen Emissionen aus Industrieprozessen, Landwirtschaft und Abfallwirtschaft. Vgl. Seite 19, Punkt 4.4:

> 80% der gesamten THG-Emissionen in Deutschland resultieren aus dem Energieverbrauch. Zu den verbleibenden 20% der THG-Emissionen tragen vor allem industrielle Prozesse, Landwirtschaft sowie Abfall (und Abwasser) bei. In kommunalen Konzepten liegt der Schwerpunkt derzeit auf der Bilanzierung energetisch-bedingter THG-Emissionen. Es gibt wenige Beispiele, bei denen die beschriebenen THG-Emissionen zusätzlich bilanziert werden. Dies liegt unter anderem an der geringen Datenverfügbarkeit auf kommunaler Ebene.

(Hier ist - leider ohne Quellenangabe - von 20% die Rede, schaut man sich die [offiziellen Zahlen des Umweltbundesamtes](https://www.umweltbundesamt.de/sites/default/files/medien/384/bilder/dateien/8_tab_thg-emi-kat_2020.pdf) über die letzten Jahre hinweg an, sind es eher ca. 15 % der Gesamtemissionen.)

Wäre also m.M.n. wesentlich sinnvoller, 15-20 % vom Budget abzuziehen. Wenn man dann alle kommunalen Budgets addiert, käme man tatsächlich in etwa beim deutschlandweiten (energiebezogenen) Gesamtbudget raus. Das ist bei 40% nicht der Fall.

**Comment by silberzwiebel** [comment]
Hmm, ich bin mir nicht sicher, ob ich Deiner Argumentation ganz folgen kann. Danke aber auf jeden Fall fürs Anstoßen!

Mein bisheriges laienhaftes Verständnis hat mich zu den (völlig groben) 40% geführt. Die Bilanz hat eben den lokalen Konsum nicht mit einbezogen und ich wollte fair sein. (Natürlich ist das für wirklich regional hergestellte Produkte etwas anderes, aber ich vermute, dass das immer noch vernachlässigbar im Vergleich zum Massenkonsum ist.)

Inwiefern man nun mit den gleichen Bilanzzahlen "nur" 15-20% vom Budget abziehen kann, leuchtet mir noch nicht ein. Oder schlägst Du vor, BISKO als _andere_ Bilanzierungsmethode zu nutzen? Ich muss gestehen, dass ich für Münster gerade gar nicht genau weiß, welche Methode genutzt wird ...
Wenn ich Dich richtig verstanden habe, würde es dann für jede Kommune individuell drauf ankommen, welche Bilanzierungsmethode genutzt wird. Entsprechend zieht man 40% oder 15-20% ab.

Nur: Meist ist es ja schon schwer, überhaupt an Daten zu kommen, da wird man sich die Methode nicht auch noch ausuchen können. Deshalb würde ich insgesamt ungerne das Fass mit unterschiedlichen Bilanzierungen pro Kommune aufmachen, weil es zu mehr Komplexität im Skript führen würde.

**Reply by jayAitch256** [reply]
Hi, um es nochmal klarer zu machen: Die 40% und die 15% sind eigentlich zwei verschiedene Punkte, die nichts miteinander zu tun haben und die ich oben eventuell etwas durcheinandergeworfen habe.

**Punkt 1: 40% abziehen / Der Konsum**

> Die Bilanz hat eben den lokalen Konsum nicht mit einbezogen und ich wollte fair sein. (Natürlich ist das für wirklich regional hergestellte Produkte etwas anderes, aber ich vermute, dass das immer noch vernachlässigbar im Vergleich zum Massenkonsum ist.)

Hmm, regionale Produkte sind ja mehr als die Erdbeere vom Erdbeerfeld nebenan, oder ;P ? Achtung, jetzt kommen meine laienhaften volkswirtschaftlichen Theorien: Jede Stadt / jede Region / jedes Land exportiert Waren (d.h. die THG Emissionen fallen in der lokalen Bilanz an) und importiert Waren (d.h. die THG Emissionen fallen woanders in der Bilanz an). In der Gesamtheit ist das dann ungefähr ausgeglichen (ja, das Gesamtkonstrukt Deutschland hat einen leichten Exportüberschuss, aber egal). Wenn du also bei allen kommunalen Budgets 40% abziehst, gehst du aber davon aus, dass alle Regionen NUR importieren und rein gar nichts exportieren (change my mind)

**Reply by jayAitch256** [reply]
**Punkt 2: 15% abziehen / die nicht energiebedingten Emissionen (Landwirtschaft, Industrieprozesse, ...)**

> Wenn ich Dich richtig verstanden habe, würde es dann für jede Kommune individuell drauf ankommen, welche Bilanzierungsmethode genutzt wird. Entsprechend zieht man 40% oder 15-20% ab.

Entweder 15-20% oder 0%, aber ja, genau. Zwar sind viele Kommunen mit BISKO oder ähnlicher Methodik bilanziert und erfassen nur die energiebedingten Emissionen, es kann aber durchaus vorkommen, dass eine Kommune mal so vorbildlich ist und die nichtenergetischen miteinrechnet. Für diesen Fall bräuchte man halt einen optionalen Parameter im Skript, der in diesen Einzelfällen verhindert, dass die 15% abgezogen werden.

Ich habe das schnell für die bisherigen Kommunen gecheckt:

BISKO bilanziert:
* Münster
* Köln
* Leipzig
* Karlsruhe (ab 2007)

Nicht direkt BISKO, aber ebenfalls nur energiebedingte Emissionen:
* Hamburg
* Landau
* Moers (Datenquelle nicht öffentlich, gezeigte Sektoren lassen es aber vermuten)
* Berlin
* München
* Düsseldorf
* Paderborn
* Dortmund
* Ulm

Auch nichtenergetische Emissionen:
* Chemnitz

Von den Kommunen, die aktuell enthalten sind, bräuchte es also nur bei Chemnitz die "Ausnahme", dass nichts vom Budget abgezogen wird. Ich persönlich halte die Implementierung für vertretbar, kann aber auch verstehen, wenn du eher gegen weitere (optionale) Parameter bist, die das Skript und die Bedienung etwas verkomplizieren.

**Reply by silberzwiebel** [reply]
Ok, hast mich überzeugt! Von mir aus können wir gerne 15-20% statt 40% abziehen. Die Diskussiongrundlage dafür steht ja hier, falls sich jemand noch tiefer in die Thematik einarbeiten will oder das irgendwie doof findet. Ich schaffe das Einarbeiten auf jeden Fall schon für Münster nicht (mehr) und noch weniger für andere Kommunen.

Die Option, nichts abzuziehen ist sicher auch nicht schwer zu implementieren. Es müsste nur sichergestellt werden, dass es pro Kommune angegeben wird, also evtl. in der .csv festhalten? Und ansonsten möchte ich halt die Hürde für neue Daten so gering wie möglich halten. Da schreckt eine Frage nach der Art der Bilanzierung potentiell ab, oder?

Perspektivisch würde ich auch das Skript noch mehr/überhaupt per CI/CD automatisieren wollen, aber erstmal steht ein Update für alle Kommunen für 2021 an.

Falls Du Zeit findest, freue ich mich über einen PR, damit bei diesem Update alles im neuen Standard ist. :) Ich selber werde frühestens im August wieder dazu kommen.

**Reply by jayAitch256** [reply]
Super! Ich denke auch, dass man die CSV Struktur weiterhin so simpel wie möglich halten sollte. Könnte mir eher vorstellen, beim Skript eine optionale Flag `--non-energetic-included` oder so anzubieten. Vielleicht komme ich in den nächsten paar Wochen dazu, einen Pull Request zu machen : )
